Analysis/src/AnalysisCategories.cpp

```diff
@@ -12,13 +12,14 @@ boost::optional<EventRegion> EventRegion::_SS_Isolated;
 boost::optional<EventRegion> EventRegion::_SS_LooseIsolated;
 boost::optional<EventRegion> EventRegion::_SS_AntiIsolated;
 
-void EventRegion::Initialize(DiscriminatorWP iso_lower, DiscriminatorWP anti_iso_lower, DiscriminatorWP anti_iso_upper)
+void EventRegion::Initialize(DiscriminatorWP iso_lower, DiscriminatorWP iso_upper,
```

Comment:
what is the purpose of `iso_upper`? I don't see any scenario when it can be useful, because POGs provide SFs only for `>=` condition...